### PR TITLE
OSDOCS#45783: Added a Release Note on CentOS 7+8 support removal

### PR DESCRIPTION
**Pull request by jherrman**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/CNV-45783

Link to docs preview:
https://84704--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by skopacz1 — COMMENTED** [review]
One comment otherwise LGTM

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -207,6 +207,11 @@ Removed features are not supported in the current release.
 //CNV-27160 Release note: REMOVED RHEL 7/virtctl RPMs
 * Installing the `virtctl` client as an RPM is no longer supported for {op-system-base-full} 7 and {op-system-base} 9.
 
+//CNV-45783
+
+* CentOS 7 and CentOS Stream 8 are now in the End of Life phase. As a consequence, the container ímages for these operating systems have been removed from OpenShift Virtualization and are no longer link:https://access.redhat.com/articles/4234591#community[community supported].
```
[the remark this answers is not in the document]
Comment (jherrman) [reply virt/release_notes/virt-4-14-release-notes.adoc:212]: Good catch, thank you!

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**